lib/aliyun/oss.rb

```diff
@@ -14,4 +14,4 @@
 require_relative 'oss/object'
 require_relative 'oss/bucket'
 require_relative 'oss/client'
-require_relative 'crcx'
+require_relative 'crcx' unless RUBY_PLATFORM =~ /java/
```

Comment:
Please add a new line here

Reply:
I hope I've added the new line as you imagined it